### ISSUE TITLE
how to handle common two or more words labels' translation? Start a relevant-short-phrases dictionary?

### ISSUE THREAD

**Issue by brazadas** [datasketch/shi18ny#9]
Sample data
Copy and paste
and common phrases in general could be in a short and relevant phrases dictionary or something like that

**Comment by jpmarindiaz** [comment]
You can just add them to the CSV dictionary and provide the translations with an id like:

sample_data
copy_paste

**Comment by brazadas** [comment]
yes but in that case it could be wearisome to search if a certain phrase _is_ in the dictionary and search its id instead of just "translating" it directly.... I think this has to do also with issue  #10; maybe this is a behaviour for another function different from i_ ... i don't know; but as I am ussing the package I would found this usefull

**Comment by jpmarindiaz** [comment]
That would make sense if we only cared about translating things from English to any other language, not from language X to language Y as we would need to provide (or guess) the original language.
So it seems indeed like it would be the job of another function to provide string translations directly, not using the id.
